README.md

```diff
@@ -1,130 +1,182 @@
 # Smartcar JS Client SDK [![Build Status][ci-image]][ci-url] [![GitHub tag][tag-image]][tag-url]
 
-The official Smartcar Javascript SDK.
+The official Smartcar Javascript SDK
 
 ## Overview
-The [Smartcar API](https://smartcar.com/docs) lets you read vehicle data (location, odometer) and send commands to vehicles (lock, unlock) to connected vehicles using HTTP requests.
 
-To make requests to a vehicle from a web or mobile application, the end user must connect their vehicle using [Smartcar's authorization flow](https://smartcar.com/docs#authentication). The Smartcar Javascript SDK provides two scripts that assist with running Smartcar's authorization flow.
-- `sdk.js`: attaches to the browser's window as `Smartcar`. Allows you to generate a Smartcar OAuth URL and launch Smartcar's authorization flow in a pop-up.
-- `callback.js`: runs on load. This file is optional and can be served in the last page of your OAuth flow to close the popup window and call the callback function that was passed in when initializing `sdk.js`.
+The [Smartcar API](https://smartcar.com/docs) lets you read vehicle data (location, odometer) and send commands (lock, unlock) to connected vehicles using HTTP requests.
```

Comment:
I think Charlotte's style guide has a blurb for Smartcar. Take a look at that so we can have the same blurb everywhere.